components/HeroUnit.js

```diff
@@ -104,3 +105,80 @@ function HeroUnitFooter () {
     </div>
   )
 }
+
+function Completionist () {
+  return (
+    <span>
+      Good luck and have fun.
+      <style jsx>{`
+        span {
+          font-size: 36px;
+        }
+        @media (max-width: 399px) {
+          span {
+            font-size: 24px;
+          }
+        }
+        `}
+      </style>
+    </span>
+  )
+}
+
+function renderer ({ days, hours, minutes, seconds, completed }) {
```

Comment:
Please rename this function to `renderCountdown` to make the name more self-descriptive.